src/patient-registration/patient-registration.component.tsx

```diff
@@ -256,18 +357,21 @@ export const PatientRegistration: React.FC = () => {
         {props => (
           <Form className={styles.form}>
             <div className={styles.formTitle}>
-              <h1 className={`omrs-type-title-1 ${styles.title}`}>New Patient</h1>
-              {localStorage.getItem('openmrs:devtools') === 'true' && <DummyDataInput setValues={props.setValues} />}
+              <h1 className={`omrs-type-title-1 ${styles.title}`}>{!!existingPatient ? 'Edit' : 'New'} Patient</h1>
```

Comment:
`!!existingPatient ? 'Edit' : 'New'` is the same as `existingPatient ? 'Edit' : 'New'`

Reply:
It's the same this actually. The valid cases to use such a pattern is when you want to assign a boolean value for a property that expects a boolean with Typescript.

It's literately coming up with an explicit expression that returns a boolean.

Reply:
But in this case is it necessary? If not then no need to have it.

Reply:
For this case it's not necessary like I mentioned above :)